src/constants/KeyCode.ts

```diff
@@ -0,0 +1,16 @@
+enum KeyCode {
```

Comment:
the lack of order here disturbs me.  you should sort this somehow

Reply:
done